src/TextareaAutosize.js

```diff
@@ -3,9 +3,11 @@
  */
 
 import React from 'react';
-import emptyFunction from 'react/lib/emptyFunction';
+import ReactDOM from 'react-dom';
```

Comment:
We could just make a new major release.
On Fri, Aug 28, 2015 at 6:12 AM Robert Fleischmann <notifications@github.com>
wrote:

> In src/TextareaAutosize.js
> <https://github.com/andreypopp/react-textarea-autosize/pull/44#discussion_r38197585>
> :
>
> > @@ -3,9 +3,11 @@
> >   */
> >
> >  import React from 'react';
> > -import emptyFunction from 'react/lib/emptyFunction';
> > +import ReactDOM from 'react-dom';
>
> Thanks @jquense <https://github.com/jquense> for the insights. So how
> should we handle this? I guess the only reasonable way to go is to wait
> until react and react-dom 0.14 is stable and then merge the PR and remove
> the support for 0.13. Any other ideas?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andreypopp/react-textarea-autosize/pull/44/files#r38197585>
> .
>